src/tests/ftest/util/test_utils_pool.py

```diff
@@ -61,6 +72,41 @@ def __init__(self, context, dmg_command, cb_handler=None):
         self.dmg = dmg_command
         self.query_data = []
 
+    def get_params(self, test):
+        """Get values for all of the command params from the yaml file.
+
+        Autosize any size/scm_size/nvme_size parameter whose value ends in "%".
+
+        Args:
+            test (Test): avocado Test object
+        """
+        super().get_params(test)
+
+        # Autosize any size/scm_size/nvme_size parameters
+        if ((self.size.value is not None and str(self.size.value).endswith("%"))
```

Comment:
(pylint-too-many-boolean-expressions) Too many boolean expressions in if statement (6/5)